tutorials/run-symfony-on-appengine-flexible/index.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Run Symfony on Google App Engine Flexible Environment
+description: Learn how to deploy a Symfony app to Google App Engine flexible environment.
+author: bshaffer
+tags: App Engine, Symfony, PHP
+date_published: 2017-03-15
+---
+## Symfony
+
+> [Symfony][symfony] is a set of PHP Components, a Web Application framework, a
+> Philosophy, and a Community — all working together in harmony.
+>
+> – symfony.com
+
+You can check out [PHP on Google Cloud Platform][php-gcp] to get an
+overview of PHP itself and learn ways to run PHP apps on Google Cloud
+Platform.
+
+## Prerequisites
+
+1. Create a project in the [Google Cloud Platform Console](https://console.cloud.google.com/).
+1. Enable billing for your project.
+1. Install the [Google Cloud SDK](https://cloud.google.com/sdk/).
+
+## Prepare
+
+1. Follow the official documentation for [installing symfony][symfony-install]
+    from symfony.com.
+
+## Run
+
+1. Run the app with the following command:
+
+        php bin/console server:run
+
+1. Visit [http://localhost:8000](http://localhost:8000) to see the Symfony
+Welcome page.
+
+## Deploy
+
+1. Create an `app.yaml` file with the following contents:
+
+        runtime: php
+        env: flex
+
+        runtime_config:
+          document_root: web
+
+1. Create an `nginx-app.conf` file with the following contents:
+
+        location / {
+          # try to serve file directly, fallback to front controller
+          try_files $uri /app.php$is_args$args;
+        }
+
+1. Add the following under "scripts" in `composer.json`:
+
+        "post-deploy-cmd": [
+            "chmod -R ug+w $APP_DIR/var"
+        ]
+
+    In the context of Symfony's `composer.json`, it will look like this:
+    ![Add post-deploy-cmd scripts to composer.json][composer-json]
+
+1. Run the following command to deploy your app:
+
+        gcloud app deploy
+
+1. Visit `http://YOUR_PROJECT_ID.appspot.com` to see the Symfony welcome page!
```

Comment:
Perhaps explicitly specify that they should replace _YOUR_PROJECT_ID_ with the project ID they created in a previous step. Presumably, they get this ID when they create a project in step #1 of Prerequisites, so maybe mention there that they should make not of their project ID.